tutorials/tts/FastPitch_MultiSpeaker_Pretraining.ipynb

```diff
@@ -0,0 +1,867 @@
+    "# Store all related text-normalized files\n",
+    "normdir = 'NeMoTTS_normalize_files'\n",
+    "# Store all supplementary files\n",
+    "suppdir = \"NeMoTTS_sup_data\"\n",
+    "# Store all config files\n",
+    "confdir = \"NeMoTTS_conf\"\n",
+    "# Store all training logs\n",
+    "logsdir = \"NeMoTTS_logs\"\n",
+    "# Store all mel-spectrograms for vocoder training\n",
+    "melsdir = \"NeMoTTS_mels\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3c46dab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import json\n",
+    "import nemo\n",
+    "import torch\n",
+    "import numpy as np\n",
+    "\n",
+    "from pathlib import Path\n",
+    "from tqdm import tqdm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c3478f9",
+   "metadata": {},
+   "source": [
+    "# 1. Pre-train FastPitch on multi-speaker data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba034e43",
+   "metadata": {},
+   "source": [
+    "## a. Dataset Preparation\n",
+    "For our tutorial, we use the subset of VCTK dataset with 5 speakers (p225-p229). The audios have 48 kHz sampling rate, we downsample to 44.1 kHz in this tutorial. \n",
+    "You can read more about dataset [here](https://datashare.ed.ac.uk/handle/10283/2950)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92ffc28d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir -p {datadir} && cd {datadir} && wget https://vctk-subset.s3.amazonaws.com/vctk_subset_multispeaker.tar.gz && tar zxf vctk_subset_multispeaker.tar.gz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7758f0a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "manidir = f\"{datadir}/vctk_subset_multispeaker\"\n",
+    "!ls {manidir}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b239e1d2",
+   "metadata": {},
+   "source": [
+    "For simplicity, we use original dev set as training set and original test set as validation set."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03c33763",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_manifest = os.path.abspath(os.path.join(manidir, 'train.json'))\n",
+    "valid_manifest = os.path.abspath(os.path.join(manidir, 'dev.json'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc0824a5",
+   "metadata": {},
+   "source": [
+    "## b. Preprocessing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3ac16425",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# additional files\n",
+    "!mkdir -p {normdir} && cd {normdir} \\\n",
+    "&& wget https://raw.githubusercontent.com/nvidia/NeMo/$BRANCH/scripts/tts_dataset_files/cmudict-0.7b_nv22.10 \\\n",
+    "&& wget https://raw.githubusercontent.com/nvidia/NeMo/$BRANCH/scripts/tts_dataset_files/heteronyms-052722 \\"
```

Comment:
Moving forward we would like tutorials to access files from a cloned version of NeMo repo, rather than wget individual files/scripts

https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/NeMo_TTS_Primer.ipynb?short_path=9930674#L46

Reply:
If user already clone the repo to use tutorial, do they still clone the whole repo again?

Reply:
Tutorials can be run in many ways. We most commonly expect them to be uploaded and run on Google Colab. But for users are who are running locally or on docker, we can ask them to comment out the "git clone" command and set NEMO_DIR to their local nemo directory.

Reply:
Add cloned version of NeMo repo.